project_wsjf/README.rst

```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Ot would me nice to use the new readme structure, that automatically generates the top level README:
https://github.com/OCA/maintainer-tools/tree/master/template/module/readme